framework/platform/bpl/uci/db/prplmesh_db

```diff
@@ -16,5 +16,5 @@ config beerocks 'config'
         option key_passphrase 'test_passphrase'
         option mem_only_psk '0'
         option backhaul_band 'auto'
-
-
+        option certification_mode '0'
```

Comment:
certification_mode should be '1'

Reply:
Please make sure this is aligned to latest UGW. Note that I changed latest UGW.

Reply:
https://git.prpl.dev/prplmesh/iwlwav/blob/iwlwav-19.07/prplmesh/files/etc/config/prplmesh